content/en/docs/contributing/development.md

```diff
@@ -26,6 +26,47 @@ cloud IDE:
 
 Gitpod automatically installs the repo-specific packages for you.
 
+## Cloning the Repository
+
+For first-time contributors, it is recommended to clone the repository using SSH instead of HTTPS to avoid common errors related to authentication.
```

Comment:
Can you elaborate what kind of issues someone might face when cloning via HTTPS and not SSH?

Reply:
Thank you @svrnm I just checked again to discover that the issue was probably a temporary one (maybe network glitch during cloning).  Cloning via HTTPS works fine .

Do you still suggest I go ahead to update the doc with instructions on cloning via SSH too or this issue can be closed?

![image](https://github.com/user-attachments/assets/df02558f-7335-4667-9d6d-0d11e645180b)

Reply:
Although I have a preference for using SSH and I would always recommend using SSH, I think for new comers choosing HTTPs might be easier to begin with. They can later upgrade. 

We might want to move this a little bit down in the document into an "Advanced" or "Additional Instructions" section. @open-telemetry/docs-approvers wdyt?

Reply:
I think that makes sense to move under an "additional instructions" section..

Yes, as an outreachy december 2024 cohort intern, I'm a new comer myself and part of the process is creating a setup for first timer contributors, so I guess this is a good add, since SSH isn't even included at all in the guide.

Should I go ahead and update accordingly with the other corrections to? @svrnm

Reply:
please go ahead, yes.